lib/widgets/compose_box.dart

```diff
@@ -667,20 +694,24 @@ class _SendButton extends StatefulWidget {
     required this.topicController,
     required this.contentController,
     required this.getDestination,
+    this.enabled = true,
   });
 
   final ComposeTopicController? topicController;
   final ComposeContentController contentController;
   final MessageDestination Function() getDestination;
+  final bool enabled;
 
   @override
   State<_SendButton> createState() => _SendButtonState();
 }
 
 class _SendButtonState extends State<_SendButton> {
   void _hasErrorsChanged() {
-    setState(() {
-      // Update disabled/non-disabled state
+    WidgetsBinding.instance.addPostFrameCallback((_) {
```

Comment:
I spotted this and wondered about it, and then happened to see something similar in another PR today and looked into it. See thread here:
https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/addPostFrameCallback/near/1893536